app/lib/Import/DataReaders/ExcelDataReader.php

```diff
@@ -123,7 +123,7 @@ public function nextRow() {
 				
 					$vn_col++;
 				
-					if ($vn_col > 255) { break; }	// max 255 columns; some Excel files have *thousands* of "phantom" columns
+					if ($vn_col > 512) { break; }	// max 255 columns; some Excel files have *thousands* of "phantom" columns
```

Comment:
I suggest adding a configuration option for this, it is a magic value that may change for unknown reasons.